cosmos-sdk-proto/Cargo.toml

```
@@ -21,13 +21,16 @@ tendermint-proto = "0.39.1"
 
 # Optional dependencies
 tonic = { version = "0.12", optional = true, default-features = false, features = ["codegen", "prost"] }
+serde = { version = "1.0.203", optional = true, default-features = false, features = ["alloc"] }
+pbjson = { package = "informalsystems-pbjson", optional = true, version = "0.7" }
 
 [features]
 default = ["grpc-transport"]
 std = ["prost/std", "tendermint-proto/std"]
 grpc = ["std", "tonic"]
 grpc-transport = ["grpc", "tonic/transport"]
 cosmwasm = []
+serialization = ["serde", "tendermint-proto/std", "pbjson"]
```

There's a lot of serialization going on in this crate, both protobuf and now JSON.

How about:

```suggestion
serde = ["dep:serde", "tendermint-proto/std", "pbjson"]
```

---

Done, and adjusted the code gen